polkadot/cli/src/cli.yml

```diff
@@ -178,3 +178,20 @@ subcommands:
               long: max-heap-pages
               value_name: COUNT
               help: The maximum number of 64KB pages to ever allocate for Wasm execution. Don't alter this unless you know what you're doing.
+  - revert:
+      about: Revert chain to the previous state
+      args:
+          - BLOCKS:
```

Comment:
maybe use `BLOCKS COUNT` or `NUM` here? I first thought, you were supposed to provide a list of block hashes or some other way you reference _a block_.